tools/Import/Importer.cs

```diff
@@ -0,0 +1,271 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Health.Fhir.Store.Utils;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.Import
+{
+    internal static class Importer
+    {
+        private static readonly HttpClient HttpClient = new HttpClient();
+        private static readonly string ConnectionString = ConfigurationManager.AppSettings["ConnectionString"];
+        private static readonly string FhirEndpoint = ConfigurationManager.AppSettings["FhirEndpoint"];
+        private static string fhirEndpoint2 = ConfigurationManager.AppSettings["FhirEndpoint2"];
+        private static readonly string ContainerName = ConfigurationManager.AppSettings["ContainerName"];
+        private static readonly int BlobRangeSize = int.Parse(ConfigurationManager.AppSettings["BlobRangeSize"]);
+        private static readonly int ReadThreads = int.Parse(ConfigurationManager.AppSettings["ReadThreads"]);
+        private static readonly int WriteThreads = int.Parse(ConfigurationManager.AppSettings["WriteThreads"]);
+        private static readonly int NumberOfBlobsToSkip = int.Parse(ConfigurationManager.AppSettings["NumberOfBlobsToSkip"]);
+        private static readonly int NumberOfBlobsToImport = int.Parse(ConfigurationManager.AppSettings["NumberOfBlobsToImport"]);
```

Comment:
To me the NumberOfBlobsToImport means how many blobs you'd import, excluding the NumberOfBlobsToSkip. I can skip 10000 blobs and import 2000 new blobs. Next time I can skip 12000 blobs and import 5000 blobs. I guess it's just a performance preference.
var globalLogPrefix = $"GlobalBlobRange=[{NumberOfBlobsToSkip + 1}-{NumberOfBlobsToImport + NumberOfBlobsToSkip}]";

Reply:
I changed name to clarify.